documents/Libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- 
```

Comment:
Maybe revert all the formatting changes (removed spaces), as it's adding noise in the diff for no real benefit. This is one of the original MaterialX library files so it will show up in the merge back to master.